src/components/MessagesList/MessagesGroup/Message/MessagePart/Poll.vue

```diff
@@ -62,6 +62,11 @@
 							{{ pollName }}
 						</h2>
 					</div>
+					<div class="poll__summary">
+						<template v-if="currentUserIsPollCreator || currentUserIsModerator || pollIsPublic">
+							{{ n('spreed', 'Poll intermediate results • %n vote', 'Poll intermediate results • %n votes', votersNumber) }}
```

Comment:
I would remove the "intermediate", if the poll is open we just simply don't append "ended"

Reply:
Okay, then https://github.com/nextcloud/spreed/pull/8270 is enough.